duckdb-vortex/src/include/vortex_error.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include "vortex.hpp"
+
+inline void HandleError(const FFIError *error) {
+	if (error != nullptr && error->code != 0) {
+		throw duckdb::InvalidInputException(error->message);
```

Comment:
Would be nice to map to diff exception down the line maybe, but def good enough as a start.

Reply:
Agree